### DISCUSSION TITLE
GraphQl Entrypoint

### DISCUSSION THREAD

**Discussion by donhill** [ExpediaGroup/graphql-kotlin#1651] [Q&A] [answered]
I am trying to setup appdynamics for the graph server. I am looking for the entry class and method that all requests hit when calling /graphql.

Can someone provide these details?

**Comment by samuelAndalon** [comment, marked as answer]
For spring -> https://github.com/ExpediaGroup/graphql-kotlin/blob/master/servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLRoutesConfiguration.kt

example for Ktor -> https://github.com/ExpediaGroup/graphql-kotlin/blob/master/examples/server/ktor-server/src/main/kotlin/com/expediagroup/graphql/examples/server/ktor/GraphQLModule.kt#L35

more information: 
https://opensource.expediagroup.com/graphql-kotlin/docs/server/spring-server/spring-overview

**Reply by donhill** [reply]
Thanks for the details!
